programs/lfx-mentorship/2024/01-Mar-May/project_ideas.md

```diff
@@ -19,3 +19,15 @@
 ## Proposed Project ideas
 
 ---
+
+### Vitess
+
+#### Improve Unit Test Coverage
+
+- Description: Vitess is a scalable cloud-native database system for horizontal scaling of MySQL.The project is over 10 years old and there are parts of the code that don’t have very good unit test coverage. Revamping these code files and adding unit test coverage will help with the overall project health. Having strong unit testing is also useful in preventing introducing bugs when making code changes to these files. The task of the mentee would be to add said unit tests for the given code files.
+- Expected Outcome: Improved unit test coverage in Vitess.
+- Recommended Skills: Go, SQL, Unit testing
+- Mentor(s): 
+  - [Manan Gupta](https://github.com/GuptaManan100) (manan@planetscale.com)
```

Comment:
Could you please add a back up mentor? This year, we really encourage having 2 mentors in all programs.

Reply:
Yes, no problem. I have added @harshit-gangal as the second mentor